### PR TITLE
added logging mechanism, rewrited LogList component, implemented logg…

### PR DESCRIPTION
**Pull request by sashka2131**
…er and small fixes

### CODE REVIEW

File: src/app/domains/Log/components/list/LogList/LogList.template.js [hunk 1]
```diff
@@ -1,35 +1,14 @@
-import { ListHOC } from 'app/components/HOCs/ListHOC'
+import List from '@material-ui/core/List'
 import { LogAdvancedView } from '../../views/LogAdvancedView'
```
Comment (okrdima) [inline src/app/domains/Log/components/list/LogList/LogList.template.js:2]: fix import

File: src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js [hunk 2]
```diff
@@ -67,21 +68,34 @@ const ProductAdvancedView = (props) => {
...
-  // [COMPONENT_STATE_HOOKS]
-  const [confirm, setConfirm] = useState(false)
-  const [deleteLoading, setDeleteLoading] = useState(false)
-
+  // [CUSTOM_HOOKS]
+  const onDeleteProductLogger = useLogger(
+    'Delete',
+    'One of products was deleted'
+  )
+  // const onMoveProductToPurchaseLogger = useLogger(
```
Comment (okrdima) [inline src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js:89]: pls fix it and remove comments

File: src/app/domains/Log/components/list/LogList/LogList.template.js (hunk 1 above)
Comment (sashka2131) [reply src/app/domains/Log/components/list/LogList/LogList.template.js:2]: what I have to fix? Everything is fine here.

File: src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js (hunk 2 above)
Comment (sashka2131) [reply src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js]: I left comment for later. It’s all fixed in the last commit

**Pull request closed without merging**